### ISSUE TITLE
scrollable-region-focusable returning false positives

### ISSUE THREAD

**Issue by nikdavy** [dequelabs/axe-core#1875]
We have multiple false positives being returned for scrollable-region-focusable returning areas which do have overflow: auto set on them. These are pre areas containing a code block; adding the overflow rule to the code element clears the problem, but I'm not entirely sure why.

Expectation: Pass the rule when the element has the required css tags
Actual: Failing the rule
Motivation: To avoid false positives blocking tests etc

<pre><code>
axe-core version: 3.4.0 (latest)
axe-webdriver, extension or other integration version: n/a
- Node version: 6.11.3
- Platform:  Mac
</code></pre>

Example:
`<pre class="prettyprint prettyprinted" data-module="app/pretty-code" style="">
    <code>
        <span class="tag">...</span>
        <span class="pln">...</span>
        <span class="atn">...</span>
        ...
    </code>
</pre>`

**Comment by straker** [comment]
Thanks for the issue. Do you have a live example we could look at? The provided code gives a rough idea of the problem but isn't triggering the failure for me.

**Comment by nikdavy** [comment]
Hi Steven,
sure, take a look at https://standards.aviva.com/framework/element-library/v4/forms/checkbox/ for example. The Chrome plugin (using Axe 3.3.2) isn't returning the errors, but our custom module, now using axe-core 3.4.0 now is. This is a new issue since we updated our axe-core version, as we haven't changed either the page content or our module since, er, a long time ago.

**Comment by nikdavy** [comment]
Been playing with this more today, and I can't get it to NOT run the rule.
`axe.run({
					runOnly : {
						type: 'tags',
						values: [ 'wcag2a', 'wcag2aa' ]
					},
					'rules' : {
						'scrollable-region-focusable' : { enabled : false }
					}
				});`
still returns all the errors.
Could you please try your end to see what happens if you attempt to turn it off? Thanks in advance...

**Comment by straker** [comment]
Very interested. Running

```js
await axe.run({
  rules: {
    "scrollable-region-focusable": { enabled: false }
  }
})
```

does not run the rule (you can see the passes array has 1 less result than a normal run)

![image](https://user-images.githubusercontent.com/2433219/67966386-6bf92880-fbc9-11e9-8b6b-bffb76ba8c06.png)

When adding tags to `runOnly` it does not run the rule either

![image](https://user-images.githubusercontent.com/2433219/67966436-80d5bc00-fbc9-11e9-91b4-99181755c78b.png)

**Comment by nikdavy** [comment]
Thanks for getting back again, but ah man, this is sooo frustrating... If I run the test via the standard Chrome plugin, I get no issues. I'm figuring it must be something to do with the test runner setup, because there'd be no other reason for variation between the flavours of axe itself. What runner are you using to generate the results above, out of interest?

**Comment by straker** [comment]
I'm using `axe-core` code injected directly into the page and outputting the results to the console. What test runner are you using and how is it set up?

**Comment by straker** [comment]
I'm going to close this since it seems to be an issue with the test runner/configuration. If you find it's something with axe please reopen.
